tempesta_fw/http.c

```diff
@@ -751,9 +751,16 @@ tfw_http_adjust_resp(TfwHttpResp *resp, TfwHttpReq *req)
 		return r;
 
 	r = tfw_http_add_hdr_via(hm);
-	if (r)
-		return r;
+	if (resp->flags & TFW_HTTP_RESP_STALE) {
+#define S_WARN_110 "Warning: 110 - Response is stale\r\n"
```

Comment:
The conventions are that headers are stored without `EOL` characters in Tempesta. A special `eolen` attribute tells the number of `EOL` bytes. So please remove `\r\n` from the string.